src/transformStateWithClones.js

```diff
@@ -65,7 +65,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const listOfTrans = [];
```

Comment:
listOfTrans? 🤔 
Probably, it's better to give it the full name `listOfTransforms`